Lib/test/test_struct.py

```diff
@@ -774,6 +812,29 @@ def test_repr(self):
         s = struct.Struct('=i2H')
         self.assertEqual(repr(s), f'Struct({s.format!r})')
 
+    def test_c_complex_round_trip(self):
+        if not have_c_complex:
```

Comment:
I would make two separate tests and use `skipIf` depending on the `have_c_complex` value. By the way, which build bots do not support float/double complex?

Reply:
> which build bots do not support float/double complex?

E.g. on Windows, WASI.